test/unittest/tests/reconciliation_tracking.cpp

```diff
@@ -0,0 +1,53 @@
+#include <catch2/catch.hpp>
+
+#include "wiredtiger.h"
+#include "wrappers/connection_wrapper.h"
+#include "wt_internal.h"
+
+TEST_CASE("ofvl_track_init", "[reconciliation]")
+{
+    auto conn = ConnectionWrapper();
+    auto session = conn.createSession();
+
```

Comment:
I'd replace this with `ConnectionWrapper conn;` as that's easier to read and more typical C++. In the current code, unless someone knows that ConnectionWrapper is a class, this could look like a function call.
I'm a big fan of auto, especially for things like STL iterators, but here, I think WT_SESSION_IMPL * would be clearer (as well as doing some type checking).
As session is a 'dumb' pointer, I suggest we should document what is happening here and why we don't need to delete it.